### PR TITLE
calendulish.SteamToolsNG version 3.0.1

### PR DESCRIPTION
**Pull request by calendulish**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101622)

### CODE REVIEW

**Comment by calendulish** [comment]
@microsoft-github-policy-service agree

**Review by stephengillie — DISMISSED** [review]

**Comment by Trenly** [comment]
[Policy] Reset Feedback

File: manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: calendulish.SteamToolsNG
+PackageVersion: latest
```
Comment (Trenly) [inline manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.installer.yaml:5]: Latest is not a valid version. The version should match what is written into control panel
```suggestion
PackageVersion: 3.0
```
After this change, the version folder will need to be renamed

File: manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: calendulish.SteamToolsNG
+PackageVersion: latest
```
Comment (Trenly) [inline manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.locale.en-US.yaml:5]: Latest is not a valid version. The version should match what is written into control panel
```suggestion
PackageVersion: 3.0
```
After this change, the version folder will need to be renamed

File: manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: calendulish.SteamToolsNG
+PackageVersion: latest
```
Comment (Trenly) [inline manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.yaml:5]: Latest is not a valid version. The version should match what is written into control panel
```suggestion
PackageVersion: 3.0
```
After this change, the version folder will need to be renamed

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
I'm not able to find the cause for this error. It installs and runs normally on a Windows 11 VM.

**Comment by stephengillie** [comment]
Automatic Validation passed with:
> Standard error:  

Yet the `Validation-Executable-Error` label persists and the waiver won't remain. Unlike similar PRs, I'm not seeing the same security check error.

**Comment by calendulish** [comment]
I'm not able to find the cause for this error as well. I have also installed it on Windows 11 Pro just fine.

```
I:\Develop\steam-tools-ng\manifests\c\calendulish\SteamToolsNG\3.0>winget install --manifest .
Found Steam Tools NG [calendulish.SteamToolsNG] Version 3.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/calendulish/steam-tools-ng/releases/latest/download/steam-tools-ng-setup.exe
  ██████████████████████████████  20.3 MB / 20.3 MB
Successfully verified installer hash
Starting package install...
Successfully installed

I:\Develop\steam-tools-ng\manifests\c\calendulish\SteamToolsNG\3.0>steam-tools-ng --version
3.0.0
```

**Review by Trenly — APPROVED** [review]

**Pull request merged**